CHANGES.rst

```diff
@@ -8,11 +8,26 @@ Unreleased
 
 * ``config_interpolation`` parameter for :class:`.Reddit` supporting basic and
   extended modes.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` contains a new parameter,
+  ``fetch``, that toggles the automatic fetching of existing data from Reddit.
+  It is set to true on default.
+* Values in methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` that are left as the
+  defaults will no longer be over-written if the ``fetch`` parameter is set to
+  true, but will fill in existing values for the flair template.
```

Comment:
This is definitely an improvement, but it changes PRAW's existing behavior, which might matter if someone was using these methods with the intention of resetting some values. This ties in with my suggestion above to make `fetch=False` the default.

Reply:
Ahh.